### PR TITLE
Add options encap-flood and description to template_bd

### PR DESCRIPTION
**Pull request by shrsr**

### CODE REVIEW

**Review by anvitha-jain — APPROVED** [review]
LGTM

File: tests/integration/targets/mso_schema_template_bd/tasks/main.yml [hunk 1]
```diff
@@ -1213,7 +1357,6 @@
     that:
     - cm_query_all_bds is not changed
     - nm_query_all_bds is not changed
-    - cm_query_all_bds.current | length == cm_query_all_bds.current | length == 3
```
Comment (lhercot) [inline tests/integration/targets/mso_schema_template_bd/tasks/main.yml:1216]: Why did we remove this check?

File: tests/integration/targets/mso_schema_template_bd/tasks/main.yml [hunk 2]
```diff
@@ -1377,77 +1484,31 @@
...
-    - query_bd_new_subnet_options.current.l2UnknownUnicast == "flood"
-    - query_bd_new_subnet_options.current.unkMcastAct == "flood"
-    - query_bd_new_subnet_options.current.multiDstPktAct == "drop"
-    - query_bd_new_subnet_options.current.l3MCast == false
-    - query_bd_new_subnet_options.current.arpFlood == true
-    - query_bd_new_subnet_options.current.v6unkMcastAct == "flood"
-    - query_bd_new_subnet_options.current.vmac == "00:00:5E:00:01:3C"
-    - query_bd_new_subnet_options.current.unicastRouting == true
-    - query_bd_new_subnet_options.current.subnets[2].primary == true
-    - query_bd_new_subnet_options.current.subnets[2].virtual == true
```
Comment (lhercot) [inline tests/integration/targets/mso_schema_template_bd/tasks/main.yml:1444]: Why did we remove and changed most of those?

**Review by lhercot — CHANGES_REQUESTED** [review]

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**